src/shared/ui/survey/StackedSlider.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, useState } from 'react';
+
+import { StackedSliderProps } from '@shared/lib';
+import { YStack, Text, SurveySlider } from '@shared/ui';
+
+const StackedSlider: FC<StackedSliderProps> = ({ config, ...props }) => {
+  const { onChange, onRelease, onPress, initialValues } = props;
+
+  const [arrayOfStackedValues, setArrayOfStackedValues] = useState(
+    initialValues || new Array(config.length).fill(null),
+  );
+
+  const onSliderValueChange = (value: number, index: number) => {
+    const clonedValues = [...arrayOfStackedValues];
+    clonedValues[index] = value;
+    setArrayOfStackedValues(clonedValues);
+    onChange(clonedValues);
+  };
+
+  const onSliderPress = () => {
+    if (!arrayOfStackedValues.some(value => value === null) && onPress) {
+      onPress();
+    }
+  };
+
+  return (
+    <YStack>
+      {config.map((sliderConfig, index) => {
+        const { sliderLabel } = sliderConfig;
+        return (
+          <YStack key={`slider-${sliderLabel}`}>
```

Comment:
It's better to set the `index` as a `key` in this case 'cause we could have the same label for many sliders